locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -527,7 +531,7 @@ public ComputeService getComputeService(Map<?,?> flags) {
 
     public ComputeService getComputeService(ConfigBag config) {
         ComputeServiceRegistry registry = getConfig(COMPUTE_SERVICE_REGISTRY);
-        return registry.findComputeService(ResolvingConfigBag.newInstanceExtending(getManagementContext(), config), true);
```

Comment:
By the time `getComputeService()` is called, I will already have set up the `ConfigBag` to be a resolving config, see `private ConfigBag getConfigBag(Map<?, ?> flags)` method below. The addition of a new `ResolvingConfigBag` was redundant. In essence I'm solving the same problem as **BROOKLYN-249** and also adding the ability to resolve config through the calling entity context, if it exists.